apps/web/src/app/(signing)/sign/[token]/date-field.tsx

```diff
@@ -16,9 +17,16 @@ import { SigningFieldContainer } from './signing-field-container';
 export type DateFieldProps = {
   field: FieldWithSignature;
   recipient: Recipient;
+  dateFormat?: string | null;
+  timezone?: string | null;
 };
 
-export const DateField = ({ field, recipient }: DateFieldProps) => {
+export const DateField = ({
+  field,
+  recipient,
+  dateFormat = 'yyyy-MM-dd hh:mm a',
+  timezone = 'Etc/UTC',
```

Comment:
Looks good, only thought is if these should live as a constant somewhere so we can use them elsewhere? I'd think so 😄 

Then we do `dateFormat = DEFAULT_DOCUMENT_DATE_FORMAT` etc.

Reply:
Oki Doki